server/routes/config/curfew.js

```diff
@@ -56,23 +72,65 @@ module.exports = {
     curfewHours: {
         licenceSection: 'curfewHours',
         fields: [
-            {daySpecificInputs: {}},
-            {allFrom: {}},
-            {allUntil: {}},
-            {mondayFrom: {}},
-            {mondayUntil: {}},
-            {tuesdayFrom: {}},
-            {tuesdayUntil: {}},
-            {wednesdayFrom: {}},
-            {wednesdayUntil: {}},
-            {thursdayFrom: {}},
-            {thursdayUntil: {}},
-            {fridayFrom: {}},
-            {fridayUntil: {}},
-            {saturdayFrom: {}},
-            {saturdayUntil: {}},
-            {sundayFrom: {}},
-            {sundayUntil: {}}
+            {daySpecificInputs: {responseType: 'optionalString'}},
+            {allFrom: {responseType: 'optionalString'}},
+            {allUntil: {responseType: 'optionalString'}},
+            {mondayFrom: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {mondayUntil: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {tuesdayFrom: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {tuesdayUntil: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {wednesdayFrom: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {wednesdayUntil: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {thursdayFrom: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {thursdayUntil: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {fridayFrom: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {fridayUntil: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {saturdayFrom: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {saturdayUntil: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {sundayFrom: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }},
+            {sundayUntil: {
+                responseType: 'requiredTime',
+                validationMessage: 'Enter a valid time'
+            }}
```

Comment:
Would it be better if there was a default message for each responseType, or is it better to have it all in one place even with the repetition?